src/generators/clientFileGenerator.ts

```diff
@@ -111,13 +110,17 @@ function writeConstructor(
       param.schemaType !== SchemaType.Constant
   );
 
+  const docs = [
+    `Initializes a new instance of the ${clientDetails.className} class.`,
+    ...requiredParams.map(p => wrapString(`@param ${p.name} ${p.description}`)),
```

Comment:
Since this is done in multiple places, it might even make sense to have a `formatJsDocParam` function that takes in a parameter and returns the wrapped doc string.